pkg/controllers/subnetset/subnetset_controller.go

```diff
@@ -79,6 +80,24 @@ func (r *SubnetSetReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 			}
 			log.V(1).Info("added finalizer on subnetset CR", "subnetset", req.NamespacedName)
 		}
+
+		// update subnetset tags if labels of namespace changed
+		nsxSubnets := r.Service.SubnetStore.GetByIndex(servicecommon.TagScopeSubnetCRType, subnet.SubnetTypeSubnetSet)
```

Comment:
can we get NSX subnets based on SubnetSet CR ID from store?

Reply:
Here, it should list all the subnetset, if it's namespace matches, then update all's tags.